index.bs

```diff
@@ -872,16 +872,21 @@ these methods.
 </pre>
 
 <div dfn-for="Transport">
-    This enumeration defines the valid transports over which a client might
-    look for an Authenticator that can provide an assertion for the respective
-    credential.  Note that these are a hint from the Relying Party to the
-    client as to its best belief of how an authenticator may be reached.
+    Non-platform Authenticators communicate with Clients using a variety of
```

Comment:
Have we defined "non-platform authenticator" up to this point? Maybe reword to be more easily understood by novice reader?